[2025-06-18 15:06:18,547] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2024-01-03T00:00:00+00:00 [queued]>
[2025-06-18 15:06:18,582] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2024-01-03T00:00:00+00:00 [queued]>
[2025-06-18 15:06:18,583] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:06:18,583] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:06:18,583] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:06:18,625] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2024-01-03T00:00:00+00:00
[2025-06-18 15:06:18,642] {standard_task_runner.py:52} INFO - Started process 11883 to run task
[2025-06-18 15:06:18,652] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'second_task', '2024-01-03T00:00:00+00:00', '--job-id', '12322', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpmg5hafe9', '--error-file', '/tmp/tmpp7r83u1v']
[2025-06-18 15:06:18,660] {standard_task_runner.py:77} INFO - Job 12322: Subtask second_task
[2025-06-18 15:06:18,898] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.second_task 2024-01-03T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:06:19,058] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=my_name
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2024-01-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-03T00:00:00+00:00
[2025-06-18 15:06:19,059] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:06:19,065] {bash.py:158} INFO - Running command: echo "this is sencond task "
[2025-06-18 15:06:19,085] {bash.py:169} INFO - Output:
[2025-06-18 15:06:19,087] {bash.py:173} INFO - this is sencond task
[2025-06-18 15:06:19,087] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:06:19,212] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, execution_date=20240103T000000, start_date=20250618T150618, end_date=20250618T150619
[2025-06-18 15:06:19,368] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:06:19,389] {local_task_job.py:146} INFO - Task exited with return code 0
